contracts/src/CrossChainApplications/ExampleMessenger/ExampleCrossChainMessenger.sol

```diff
@@ -124,6 +124,23 @@ contract ExampleCrossChainMessenger is
             );
     }
 
+    /**
+     * @dev See {TeleporterUpgradeable-updateMinTeleporterVersion}
+     *
+     * Updates the minimum Teleporter version allowed for receiving on this contract
+     * to the latest version registered in the {TeleporterRegistry}.
+     *
+     * Emits a {MinTeleporterVersionUpdated} event.
+     */
+    function updateMinTeleporterVersion() external override {
```

Comment:
Do we want to make this `onlyOwner` as well? If not, we should explain the use case for why not.

Reply:
added to documentation noting this is an example cross chain messenger, and that anyone would be able to call this function, disallowing receiving of old Teleporter version messages.

Reply:
I generally think we should avoid using any patterns in the example that we wouldn't encourage others to use for real projects. Allowing anyone to call this method is potentially dangerous, so would heir on the side of allowing no one to call it, or it being `onlyOwner`

Reply:
agreed, changed `ExampleCrossChainMessenger` to be ownable as well